client/src/components/Minimap/NodeComponent.tsx

```diff
@@ -160,7 +162,8 @@ const NodeComponent = ({
           showArrow={
```

Comment:
Is there a way we can simplify the first prop in to 1 variable because the `showArrow` prop here and the one above seem identical along with the className